react-app/src/library/services/headset.ts

```diff
@@ -301,6 +301,8 @@ export default class HeadsetService {
     const implementation = this.getConnectedImpl();
     if (implementation) {
       return implementation.resetHeadsetStateForCall(conversationId);
+    } else {
+      this.logger.info('No active implementation, headset state does not require a reset');
```

Comment:
I noticed that this file was no longer at 100% coverage and this was the culprit.  I added this `else` block primarily so I could have something to check against when we don't have an implementation present because without an implementation present I don't think we would be able to mock a function that exists in said implementation.  I also think maybe this log isn't the _worst_ thing to have.